spec/lib/appsignal_reporting_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.xdescribe AppsignalReporting do
```

Comment:
Did you mean to disable this group of specs?

Reply:
Yes, I'm considering it a TODO